### DISCUSSION TITLE
A big problem!

### DISCUSSION THREAD

**Discussion by alex28282828** [arkypita/LaserGRBL#1589] [Q&A]
Who can help me with this? 

When I engrave a proyect, pass that. The borde engrave has a problem...
![WhatsApp Image 2021-09-26 at 3 20 16 PM](https://user-images.githubusercontent.com/91433552/134821621-bf3c983d-5f95-4b57-a343-bfaeae765e3a.jpeg)

**Comment by Nalofox** [comment]
Hi,

Do you have solved your problem ?

**Comment by dungjk** [comment]
It may cause by backslash.
Check belt tension, bearing, motors,...
